lib/invisible_captcha.rb

```diff
@@ -1,12 +1,15 @@
 # frozen_string_literal: true
-
+require 'rails'
+require 'responders'
```

Comment:
those 2 requires are unnecessary, AFAIK `responders` is not a Rails dependency, so this will crash as you can see in the CI https://travis-ci.com/github/markets/invisible_captcha/jobs/481937373#L320:

```
Failure/Error: require 'responders'
```